content/docs/csidriver/features/powerflex.md

```diff
@@ -385,6 +385,16 @@ The CSI PowerFlex driver version 1.3 and later support the automatic deployment
   Refer to https://hub.docker.com/r/dellemc/sdc for supported OS versions.
 - There is no automated uninstallation of the SDC kernel module. Follow PowerFlex SDC documentation to manually uninstall the SDC driver from the node. 
 
+Starting from CSM 1.12.0, automatic deployment of the SDC can be disabled. If you have a Powerflex 4.x and NAS functionality enabled, SDC deployment is not necessary. By default, deployment of SDC will be enabled but to disable it, set the `sdc` field to `false`.
```

Comment:
"From CSM 1.12.0, you can disable automatic SDC deployment.

If you have PowerFlex 4.x with NAS enabled, SDC deployment isn’t needed. By default, SDC deployment is enabled. To disable it, set the sdc field to false."